lace_grid.py

```diff
@@ -13,131 +13,152 @@
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see http://www.gnu.org/licenses/.
 
-from math import *
-
-# These lines are only needed if you don't put the script directly into
-# the installation directory
 import sys
-from os import path

```

Comment:
Much better indeed, I adjusted the [readme](https://github.com/d-bl/inkscape-bobbinlace#inkscape-bobbinlace). That is one page down for issue #1.